### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Dreyrk**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .App {
```
Comment (cyrheel) [inline src/App.css:1]: move your css file into a Style folder (inside src folder oc)

File: src/Pages/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    url: "/api/cards",
+    baseURL: "http://localhost:5000",
+    headers: {
+      "Access-Control-Allow-Origin": "*",
+    },
+  })
+    .then((res) => res.data)
+    .then((data) => {
+      setCardsData(data);
+      console.log(cardsData);
```
Comment (cyrheel) [inline src/Pages/Home.jsx:28]: avoid console.log() except for console.log(error)

File: src/components/Cards.jsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        <TitleRarityContainer>
+          <CardTitle>
+            <h1>{Name}</h1>
+          </CardTitle>
+          <RarityContainer>{Rarity}</RarityContainer>
+        </TitleRarityContainer>
+        <ImgContainer src="public\src\back-card-yugioh.png"></ImgContainer>
+        <DescContainer>{Description}</DescContainer>
+      </CardContainer>
+      <CardBack></CardBack>
```
Comment (cyrheel) [inline src/components/Cards.jsx:77]: use syntaxt </> not <></>

File: src/components/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import logo from "../../public/src/logo";
+
+function Header() {
+  return (
+    <div>
+      <NavLink to="/">
```
Comment (cyrheel) [inline src/components/Header.jsx:8]: search for this : 

const navigate = useNavigate();

onClick() => navigate('/', {replace: true});

File: src/index.css [hunk 5]
```diff
@@ -1,13 +1,13 @@
 body {
```
Comment (cyrheel) [inline src/index.css:1]: move css file into style folder

**Review by cyrheel — CHANGES_REQUESTED** [review]

File: src/components/Cards.jsx (hunk 3 above)
Comment (Dreyrk) [reply src/components/Cards.jsx:77]: i'm going to put children in

**Review by cyrheel — APPROVED** [review]

**Pull request merged**